edubots_surveys.Rnw

```diff
@@ -341,128 +341,126 @@ norms for these MIM groups in order to maintain the safety of these online
+conceptualizing learners' questions could help designers integrate the
+appropriate types of questions that chatbots should support for different
+courses. Other authors, such as \cite{coronado2018cognitive}, have proposed agents that
+store learning materials to be provided on demand to students, while \cite{crockett2017predicting} 
+reported on tutoring systems that can
 perform initial assessments of students' understanding and provide learning
-material that would advance their understanding to the next level.
+materials to advance their understanding to the next level.
 
 Regarding the factors for the adoption of chatbots in higher education, many
 studies have focused on the evaluation of technology acceptance and usability
 \cite{Roblyer2010,Pimmer2019}. However, higher education is a special domain
 where, according to \cite{hobert2019you}, specific pedagogical factors
 such as learning success and increased motivation are more important.  Therefore,
-to develop \textcolor{black}{effective chatbots for higher education, all stakeholders'
-(i.e. educators, students, institutions, etc) needs should be carefully
+to develop effective chatbots for higher education, the needs of all stakeholders
+(i.e. educators, students, institutions, etc) should be carefully
 collected and taken into consideration
-\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student's learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance; this is why we focus in this paper on what the teachers are looking for in a TAM  (Technology Acceptance Model) and associated technologies like chatbots; both authors focus on what the teachers need in terms of the latter; this paper will focus on a wider perspective trying to ascertain what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned in to the user needs and experience. In this paper we will try to find out what those are in order to propose a successful model of adoption of chatbot technology}.
+\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. This is why we focus in this paper on what  teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```

Comment:
I don't really get this:

> This is why we focus in this paper on what teachers are looking for in a Technology Acceptance Model (TAM) and associated technologies like chatbots.

Do we really use a Technology Acceptance Model? "We are looking for... associated technologies like chatbots"?
Probably delete that whole sentence.

```suggestion
\cite{sjostrom2018designing,tsivitanidouusers}. These needs include, but are not limited to, student learning success due to higher motivation, but these are a posteriori effects that cannot be assessed in advance. Both authors focus on what teachers need in terms of the latter; this paper will focus on a wider perspective, trying to determine what they are looking for in terms of general messaging technology. The classical literature \cite{tam91} already proposes that any adoption of technology must be tuned to the user needs and experience. In this paper, we will try to find out what those are in order to propose a successful model of adoption of chatbot technology.
```